app/infrastructure/clients/barun_client.py

```
@@ -29,20 +29,20 @@ async def login(self) -> aiohttp.ClientSession:
                 allow_redirects=True,
             ) as login_response:
                 if login_response.status != 200:
-                    raise ClientException(error_code=ErrorCode.BARUN_ERROR)
+                    raise AppException(error_code=ErrorCode.BARUN_ERROR)
```

Ruff (0.14.7)

Abstract `raise` to an inner function

(TRY301)